app/Http/Backend/Requests/AddCategoryRequest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Http\Backend\Requests;
+
+use App\Http\Backend\Requests\Request;
+use App\Http\Backend\Requests\CategoryRequest;
+
+class CategoryRequest extends Request
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     **/
+    public function rules()
+    {
+        return [
+            'txtName'=>'required|unique:categories,name',
+            'image'=>'required|image|max:2048',
+        ];
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     **/
+    public function messages()
+    {
+        return [
+            'txtName.required'=>'Vui lòng nhập tên chuyên mục !',
```

Comment:
use this open-source instead of defining validation message. 
https://github.com/caouecs/Laravel-lang/tree/master/vi